

### Suman runner start ###

Beginning of run at 1465885582566 = [Mon Jun 13 2016 23:26:22 GMT-0700 (PDT)]
Command issued from the following directory "/Users/Olegzandr/WebstormProjects/poolio"
Command = ["/Users/Olegzandr/.nvm/versions/node/v6.2.1/bin/node","/Users/Olegzandr/.nvm/versions/node/v6.2.1/bin/suman","-v"]


[test-repeated-inits.js] 
[test-repeated-inits.js] 
[test-repeated-inits.js]  ### Suman end run ### 
[test-repeated-inits.js] 
[test-repeated-inits.js] 
[test-repeated-inits.js] 
[test-repeated-inits.js] 

[test.js]  => Poolio: your worker pool experienced an error =>  { msg: 'error',
[test.js]   error: '[some user error]',
[test.js]   result: null,
[test.js]   workId: 1 }
[test.js] 

[test.js]  => Poolio => please add your own "error" event listener using pool.on("error", fn) to prevent these error messages from being logged.
[test.js] 

[test.js]  => Poolio: your worker pool experienced an error =>  { msg: 'error', error: 'beetles', result: null, workId: 1 }
[test.js]  => Poolio => please add your own "error" event listener using pool.on("error", fn) to prevent these error messages from being logged.
[test.js] 

[test-hook-params.js] TypeError: Converting circular structure to JSON
[test-hook-params.js]     at Object.stringify (native)
[test-hook-params.js]     at Suman.logData (/Users/Olegzandr/WebstormProjects/suman/lib/suman.js:492:29)
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/lib/exec-suite.js:358:13
[test-hook-params.js]     at allDone (/Users/Olegzandr/WebstormProjects/suman/lib/TestSuite.js:680:6)
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:726:13
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:52:16
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:269:32
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:44:16
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:723:17
[test-hook-params.js]     at /Users/Olegzandr/WebstormProjects/suman/node_modules/async/lib/async.js:167:37
[test-hook-params.js] 

[test-hook-params.js] 
[test-hook-params.js] 
[test-hook-params.js]  ### Suman end run ### 
[test-hook-params.js] 
[test-hook-params.js] 
[test-hook-params.js] 
[test-hook-params.js] 

[test.js]  => Poolio: your worker pool experienced an error =>  { msg: 'error', error: 'beetles', result: null, workId: 2 }
[test.js]  => Poolio => please add your own "error" event listener using pool.on("error", fn) to prevent these error messages from being logged.
[test.js] 

[test.js] 
[test.js] 
[test.js]  ### Suman end run ### 
[test.js] 
[test.js] 
[test.js] 
[test.js] 

[test-canonical.js] 
[test-canonical.js] 
[test-canonical.js]  ### Suman end run ### 
[test-canonical.js] 
[test-canonical.js] 
[test-canonical.js] 
[test-canonical.js] 